docs/gallery_code/meteorology/plot_zonal_mean.py

```diff
@@ -0,0 +1,75 @@
+"""
+Zonal Mean Diagram of Air Temperature
+=====================================
+
+This example demonstrates aligning a linear plot and a cartographic plot using Matplotlib.
+
+"""
+
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+from mpl_toolkits.axes_grid1 import make_axes_locatable
+import numpy as np
+
+import iris
+from iris.analysis import MEAN
+import iris.plot as iplt
+
+
+def main():
+    fname = iris.sample_data_path("air_temp.pp")
+    temperature = iris.load_cube(fname)
+    collapsed_temp = temperature.collapsed("longitude", MEAN)
+
+    # Set y axes with -90 and 90 limits and spacing of 15 per tick.
+    yticks = np.arange(-90, 105, 15)
+    ylim = [-90, 90]
+    fig = plt.figure(figsize=[12, 4])
+    ax1 = fig.add_subplot(111, projection=ccrs.PlateCarree())
+    plt.sca(ax1)
+    im = iplt.contourf(temperature, cmap="RdYlBu_r")
+    ax1.coastlines()
+    ax1.gridlines()
+    ax1.set_xticks([-180, -90, 0, 90, 180], crs=ccrs.PlateCarree())
+    ax1.set_yticks(yticks, crs=ccrs.PlateCarree())
+    ax1.set_title("Air Temperature")
+    ax1.set_ylabel("latitude")
+    ax1.set_xlabel("longitude")
+    ax1.set_ylim(*ylim)
+    divider = make_axes_locatable(ax1)
```

Comment:
This is the 'magic sauce' that makes it possible to align everything, so it should have some extra emphasis. I've done a bit of reading to work out an appropriate comment:

```suggestion
    
    # Create a Matplotlib AxesDivider object to allow alignment of other Axes objects.
    divider = make_axes_locatable(ax1)
```